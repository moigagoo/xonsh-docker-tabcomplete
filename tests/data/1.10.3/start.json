{
        "arguments": [
                "-a", "--attach",
                "--detach-keys",
                "--help",
                "-i", "--interactive"
        ],
        "commands": []
}
