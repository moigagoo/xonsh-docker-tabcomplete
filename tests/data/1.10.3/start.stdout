
Usage:	docker start [OPTIONS] CONTAINER [CONTAINER...]

Start one or more stopped containers

  -a, --attach         Attach STDOUT/STDERR and forward signals
  --detach-keys        Override the key sequence for detaching a container
  --help               Print usage
  -i, --interactive    Attach container's STDIN
